plugin/airline.vim

```diff
@@ -26,6 +26,18 @@ function! s:init()
   silent doautocmd User AirlineAfterInit
 endfunction
 
+function! s:update_tabline()
+  if pumvisible()
+    return
+  elseif !get(g:, 'airline#extensions#tabline#enabled', 0)
+    return
+  endif
+  call s:init()
```

Comment:
the call to `s:init()` seems to be the only reason that this code is here instead of the extension.  can the autocmd not be set up later inside the tabline extension?